src/python/grpcio/grpc/__init__.py

```diff
@@ -413,6 +414,8 @@ class ClientCallDetails(six.with_metaclass(abc.ABCMeta)):
       credentials: An optional CallCredentials for the RPC.
       wait_for_ready: This is an EXPERIMENTAL argument. An optional flag t
         enable wait for ready mechanism.
+      compression: An element of grpc.compression, e.g.
+        grpc.compression.Gzip.
```

Comment:
Will these APIs be labeled as experimental for now?

Reply:
That was not my plan. If you feel strongly about it, though, we can mark them as experimental.